template.json

```diff
@@ -38,6 +38,7 @@
       "@types/react-router-dom": "^5.3.1",
       "@types/webpack-env": "1.18.0",
       "@types/jest": "^29.5.1",
+      "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
Why is this needed?

Reply:
I ran into this. Basically CRA is dead https://github.com/facebook/create-react-app/issues/13216

Reply:
Hmm someone fixed it here: https://github.com/facebook/create-react-app/commit/0f5e990b8a04f53861d64ff53751517bbf73d867

But looks like no new versions have published in a while.